spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/functions/SparkFunctions.java

```diff
@@ -30,14 +30,19 @@ public class SparkFunctions {
   private SparkFunctions() {}
 
   private static final Map<String, UnboundFunction> FUNCTIONS =
-      ImmutableMap.of(
-          "iceberg_version", new IcebergVersionFunction(),
-          "years", new YearsFunction(),
-          "months", new MonthsFunction(),
-          "days", new DaysFunction(),
-          "hours", new HoursFunction(),
-          "bucket", new BucketFunction(),
-          "truncate", new TruncateFunction());
+      new ImmutableMap.Builder<String, UnboundFunction>()
+          .put("iceberg_version", new IcebergVersionFunction())
+          .put("years", new YearsFunction())
+          .put("year", new YearsFunction())
+          .put("months", new MonthsFunction())
+          .put("month", new MonthsFunction())
+          .put("days", new DaysFunction())
+          .put("day", new DaysFunction())
+          .put("hours", new HoursFunction())
+          .put("hour", new HoursFunction())
```

Comment:
These function classes contain examples with plural styles in javadoc. Can we add another example or just update them?

Reply:
That's a good point. Let me update the javadoc of those functions to add examples with the singular form.
The functions also have `name()` methods that return the plural form. I'm not sure if we should change the name though. For now, my thinking is to leave the name alone, but just to support using the singular form (and documenting the use in javadoc). If there is a doc page that should be updated, I'll be happy to update it if you let me know.